R/pkg/inst/tests/testthat/test_context.R

```diff
@@ -177,6 +177,13 @@ test_that("add and get file to be downloaded with Spark job on every node", {
   spark.addFile(path)
   download_path <- spark.getSparkFiles(filename)
   expect_equal(readLines(download_path), words)
+
+  # Test spark.getSparkFiles works well on executors.
+  seq <- seq(from = 1, to = 10, length.out = 5)
+  f <- function(seq) { readLines(spark.getSparkFiles(filename)) }
+  results <- spark.lapply(seq, f)
+  for (i in 1:5) { expect_equal(results[[i]], words) }
+
```

Comment:
* It fails when I run ```run-tests.sh``` on my machine.
* It succeed when I paste these code to SparkR console.
* It succeed when I paste these code to text file, and submit by ```bin/spark-submit test.R```(local mode) or ```bin/spark-submit --master yarn test.R```(yarn mode).

So I think it may caused by the test suites infrastructure, but I'm not familiar with that part.